content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -876,7 +876,7 @@ it won't be supported in a future Kubernetes release.
 
 #### Retroactive default StorageClass assignment
 
-{{< feature-state for_k8s_version="v1.26" state="beta" >}}
+{{< feature-state for_k8s_version="v1.28" state="stable" >}}
```

Comment:
Hi @RomanBednar 
The `RetroactiveDefaultStorageClass` feature gate will beGA in k8sv1.28 but if you change it here it will reflect on the current website (content according to the current release i.e. k8 v1.27).
So, please target these changes at the `dev-1.28` branch, which will become the live website once after k8s v1.28 will release.

Reply:
Ack, forgot again. Thanks!